cmake/InstallMongo.cmake

```diff
@@ -55,6 +57,7 @@ if(NOT "${INSTALL_MONGO_RET}" STREQUAL "0")
     message(FATAL_ERROR "Error when building and installing mongoc, see more in log ${INSTALL_MONGO_LOG}")
 endif()
 
+message(STATUS "Running ${CMAKE_COMMAND}")
```

Comment:
Are these debug statements suppsoed to be still there?

Reply:
pushed a removal/revert